src/api/process.c

```diff
@@ -117,7 +117,7 @@ static bool signal_process(process_t* proc, signal_e sig) {
 
 static int process_start(lua_State* L) {
   size_t env_len = 0, key_len, val_len;
-  const char *cmd[256], *env_names[256] = { NULL }, *env_values[256] = { NULL }, *cwd = NULL;
+  const char *cmd[256] = { NULL }, *env_names[256] = { NULL }, *env_values[256] = { NULL }, *cwd = NULL;
```

Comment:
initialize array with default values so we don't have to worry about unexpected surprises